src/vehicle.ts

```diff
@@ -14,72 +14,25 @@ export class Vehicle {
       }
       this[name] = this[name].bind(this);
     }
-
-    // TODO @Shinigami92 2022-01-13: Find better strategy
-    // @ts-expect-error
-    this.vehicle.schema = {
-      description: 'Generates a random vehicle.',
-      sampleResults: ['BMW Explorer', 'Ford Camry', 'Lamborghini Ranchero'],
-    };
-    // @ts-expect-error
-    this.manufacturer.schema = {
-      description: 'Generates a manufacturer name.',
-      sampleResults: ['Ford', 'Jeep', 'Tesla'],
-    };
-    // @ts-expect-error
-    this.model.schema = {
-      description: 'Generates a vehicle model.',
-      sampleResults: ['Explorer', 'Camry', 'Ranchero'],
-    };
-    // @ts-expect-error
-    this.type.schema = {
-      description: 'Generates a vehicle type.',
-      sampleResults: ['Coupe', 'Convertable', 'Sedan', 'SUV'],
-    };
-    // @ts-expect-error
-    this.fuel.schema = {
-      description: 'Generates a fuel type.',
-      sampleResults: ['Electric', 'Gasoline', 'Diesel'],
-    };
-    // @ts-expect-error
-    this.vin.schema = {
-      description: 'Generates a valid VIN number.',
-      sampleResults: ['YV1MH682762184654', '3C7WRMBJ2EG208836'],
-    };
-    // @ts-expect-error
-    this.color.schema = {
-      description: 'Generates a color',
-      sampleResults: ['red', 'white', 'black'],
-    };
-    // @ts-expect-error
-    this.vrm.schema = {
-      description: 'Generates a vehicle vrm',
-      sampleResults: ['MF56UPA', 'GL19AAQ', 'SF20TTA'],
-    };
-    // @ts-expect-error
-    this.bicycle.schema = {
-      description: 'Generates a type of bicycle',
-      sampleResults: [
-        'Adventure Road Bicycle',
-        'City Bicycle',
-        'Recumbent Bicycle',
-      ],
-    };
   }
 
   /**
-   * vehicle
+   * Returns a random vehicle.
+   * @returns {string}
    *
-   * @method faker.vehicle.vehicle
+   * @example
+   * 'BMW Explorer', 'Ford Camry', 'Lamborghini Ranchero'
```

Comment:
Please reformat these to

```suggestion
   * @example 'BMW Explorer', 'Ford Camry', 'Lamborghini Ranchero'
```

Reply:
Done